int32 force_threshold
---
bool is_grabbing
